kivy/uix/textinput.py

```diff
@@ -1487,21 +1487,25 @@ def cancel_long_touch_event(self):
             self._long_touch_ev.cancel()
             self._long_touch_ev = None
 
+    def _select_word(self, delimiters=u' .,:;!?\'"<>()[]{}'):
+        ci = int(self.cursor_index())
+        cc = int(self.cursor_col)
+        line = self._lines[self.cursor_row]
+        len_line = len(line)
```

Comment:
Again, not your fault, but `len_line` is just used once, how about doing some cleanup?